src/shared/__tests__/libs/i18n.spec.js

```diff
@@ -0,0 +1,57 @@
+import { expect } from 'chai';
+import { getLabelFromLocale, getLocaleFromLabel, detectLocale } from '../../libs/i18n';
+
+describe('libs: i18n', () => {
+    describe('#getLabelFromLocale', () => {
+        describe('when a supported locale code is passed', () => {
+            it('should return the correct label', () => {
+                const locale = 'de';
+                expect(getLabelFromLocale(locale)).to.equal('Deutsch - German');
+            });
+        });
+
+        describe('when an unsupported locale is passed', () => {
+            // FIXME: Prettier is using double quotes for strings with escaped single quotes
+            //eslint-disable-next-line quotes
+            it("should return 'English (International)'", () => {
```

Comment:
`it('should return "English (International)"')` doesn't work?

Reply:
Ooh, I'll try that. I only tried `it('should return \'English (International)\''`